### DISCUSSION TITLE
[Feature Request] Notify component

### DISCUSSION THREAD

**Discussion by JulianGarcia04** [heroui-inc/heroui#1974] [Feature requests]
### Is your feature request related to a problem? Please describe.

This problem arose when is creating a component that did a data fetching and i needed a little modal that showed when had a error, or showed a confirm message, but NEXTUI don't has a component

### Describe the solution you'd like

Bro you can follow the solution as is doing Quasar framework from Vue.

I give you the docuementationd where is the form use and more...

https://quasar.dev/quasar-plugins/notify

### Describe alternatives you've considered

Is the same to the previus field

### Screenshots or Videos

![Screenshot 2023-11-14 at 6 14 33 AM](https://github.com/nextui-org/nextui/assets/86687709/763922ed-99da-487f-b68d-0740c1b045f5)
![Screenshot 2023-11-14 at 6 15 16 AM](https://github.com/nextui-org/nextui/assets/86687709/79886359-86b9-4aa6-ae91-74d540c0c2f1)

**Comment by yWilliamLima** [comment]
up

**Comment by wingkwong** [comment]
see #314
